packages/neon/neon_files/pubspec.yaml

```diff
@@ -36,6 +36,7 @@ dependencies:
   queue: ^3.0.0
   rxdart: ^0.27.0
   share_plus: ^7.0.0
+  timezone: ^0.9.2
```

Comment:
0.9.0 should work as well

Reply:
I'd rather stay with the up to date timezone data.
I want to go with the default timezone db with no historic data and 5years worth of future data.
0.9.0 has been released in 2022-08 so we'd only have another 2.5 years worth of future data.

0.9.2 was released 20 months ago so I don't think it will restrain any reasonably up to date dependencies.